Apps/Patient/src/Controllers/PatientController.cs

```diff
@@ -86,6 +86,7 @@ public async Task<RequestResult<PatientModel>> GetPatient(string hdid)
         /// The client does not have access rights to the content; that is, it is unauthorized, so the server
         /// is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.
         /// </response>
+        /// <response code="502">Unable to get response from client registry.</response>
```

Comment:
I think a `[ProducesResponseType]` annotation should be added to mirror this.

Reply:
It's already there isn't it?  
<img width="2533" alt="Screenshot 2022-11-10 at 9 49 40 AM" src="https://user-images.githubusercontent.com/58790456/201169759-61a1fb77-14af-4fe0-bc84-7f3a5e8cbacc.png">